lib/book.rb

```diff
@@ -0,0 +1,18 @@
+class Book
```

Comment:
I really love how you are using methods to reduce the storage load of your instance variables!